### PR TITLE
add two QoL unit widgets

### PR DESCRIPTION
**Pull request by verybadsoldier**
Please lemme have these before custom widgets get banned ;) I would be able to play the game then with disabled user widgets

## Unit Groups - No camera move on group select

Disables camera movement to unit group when double-pressing the unit group number on keyboard. Defaults to `off`

## Unit Groups - Remove units from a group

Implements action `groupclear` to remove units from their current group. I think this is missing in general. Defaults to `on`

Thanks guys ;)

### CODE REVIEW

File: luaui/Widgets/cmd_unitgroup_no_camera_move_on_group_select.lua [hunk 1]
```diff
@@ -0,0 +1,20 @@
+local versionNumber = "1.0"
+
+function widget:GetInfo()
+	return {
+		name      = "Unit Groups - No camera move on group select",
```
Comment (sprunk) [inline luaui/Widgets/cmd_unitgroup_no_camera_move_on_group_select.lua:5]: can't you just unbind groupmove instead?

File: luaui/Widgets/cmd_unitgroup_remove.lua [hunk 2]
```diff
@@ -0,0 +1,28 @@
+function widget:GetInfo()
+  return {
+    name      = "Unit Groups - Remove units from a group",
+    desc      = "Remove units from a unit group on action 'groupclear'",
```
Comment (nbusseneau) [inline luaui/Widgets/cmd_unitgroup_remove.lua:4]: Isn't this already covered by CTRL+`? I use it all the time to remove selected units from their current group.

**Comment by WatchTheFort** [comment]
Why is the first behaviour unwanted? Single tap to select, double tap to select and centre should be the universal behaviour of all selection functions.

File: luaui/Widgets/cmd_unitgroup_remove.lua (hunk 2 above)
Comment (sprunk) [reply luaui/Widgets/cmd_unitgroup_remove.lua:4]: Yeah you're binding the wrong action. It's not `groupclear` it's `group unset`, and it's already bound by default
https://github.com/beyond-all-reason/Beyond-All-Reason/blob/295cf560c8f7c8b751ee0fd13ec19549061441b6/luaui/configs/hotkeys/default_keys.txt#L311
https://github.com/beyond-all-reason/Beyond-All-Reason/blob/295cf560c8f7c8b751ee0fd13ec19549061441b6/luaui/configs/hotkeys/grid_keys.txt#L154

**Review by sprunk — CHANGES_REQUESTED** [review]

File: luaui/Widgets/cmd_unitgroup_remove.lua (hunk 2 above)
Comment (verybadsoldier) [reply luaui/Widgets/cmd_unitgroup_remove.lua:4]: Okay thanks, that works. For reference:
https://springrts.com/phpbb/viewtopic.php?f=1&t=44462

Will remove the widget 👍

**Comment by verybadsoldier** [comment]
> Why is the first behaviour unwanted? Single tap to select, double tap to select and centre should be the universal behaviour of all selection functions.

Cause some people (aka me) don't want to trigger it by accident.

**Review by sprunk — COMMENTED** [review]
Ideally the "move to group" behaviour would be optional in the engine

**Review by sprunk — CHANGES_REQUESTED** [review]
Change default BAR bindings from
```
bind 1 group 
// group 2, etc
``` 
to
```
bind 1,1 group  focus 1
bind   1 group select 1
// group 2, etc
``` 
and then you don't need a widget, just edit keybindings.

See also https://github.com/beyond-all-reason/spring/pull/793 and #1985

**Comment by verybadsoldier** [comment]
Since sprunk solved everything, no widgets needed anymore. Case closed. Thanks!

**Pull request closed without merging**